app/protocols/index.js

```diff
@@ -67,6 +69,7 @@ module.exports = {
 function registerPriviledges () {
   globalProtocol.registerSchemesAsPrivileged([
     { scheme: 'hyper', privileges: P2P_PRIVILEDGES },
+    { scheme: 'ssb', privileges: { ...P2P_PRIVILEDGES, standard: false } }, // ssb uri are not url safe
```

Comment:
Mind splitting this out into its own `SSB_PRIVILEDGES` const at the top instead of doing it inline?

Reply:
Disregard this comment, instead get rid of the `standard:false` bit and instead use `P2P_PRIVILEDGES` directly. (as per https://github.com/AgregoreWeb/agregore-browser/pull/144#issuecomment-1005071884

Reply:
Ah yes, that's fixed it! Thanks :) I hope you don't mind me being nitpicky too because I've corrected the spelling of privileges :rofl: :pray: